.changelog/release-v15.2.0.md

```diff
@@ -0,0 +1,157 @@
+# Helm Release Notes
+
+Date | Revision | Description
+---------|----------|---------
+ 2023-10-23 | 0 | Initial draft
+ 2023-10-31 | 1 | RC.1
+
+- For *BREAKING CHANGES*, please review the section `#5` "Breaking Changes" below.
+- For *KNOWN ISSUES*, please review the section `#8` "Known Issues" below.
+
+## 0. Summary
+
+Enhancements and non-breaking changes to the [v15.1.0 Release](https://github.com/mojaloop/helm/blob/master/.changelog/release-v15.1.0.md), which includes:
+
+1. Nodejs version of core and supporting services upgraded to v18.17.1 LTS
+2. Performance improvements and metrics around core services: ALS, Central Ledger, Quoting Service based on characterisation and fixes, enhancements done
+3. General maintenance and bug fixes
+
+## 1. New Features
+
+1. **Performance fixes, enhancements and maintenance:**  
```

Comment:
I just noticed that some of these updates were missed/swallowed: 

- mojaloop/#3519: add topic name override config map logic (https://github.com/mojaloop/central-ledger/issues/969) ([b99d091](https://github.com/mojaloop/central-ledger/commit/b99d09129f328aab60e4059035be4d263ecd9f98)), closes https://github.com/mojaloop/project/issues/3519 - https://github.com/mojaloop/central-ledger/releases/tag/v17.2.0
- mojaloop/#3498: alter message keys for prepare, fulfil and timeout (https://github.com/mojaloop/central-ledger/issues/965) ([db3e3b2](https://github.com/mojaloop/central-ledger/commit/db3e3b2f0450f1093f1f20eb4f49b4f4de53ddd0)), closes https://github.com/mojaloop/project/issues/3498 - from https://github.com/mojaloop/central-ledger/releases/tag/v17.1.0
- json stringify on every response (https://github.com/mojaloop/central-ledger/issues/961) ([6798bfd](https://github.com/mojaloop/central-ledger/commit/6798bfda51be18c2d44c22e10ef67bc76c8b2195)) - https://github.com/mojaloop/central-ledger/releases/tag/v17.0.5
- mojaloop/#2880: simplify liquidity cover check and fix issue with validation (for main) (https://github.com/mojaloop/central-ledger/issues/918) ([825faf7](https://github.com/mojaloop/central-ledger/commit/825faf7c1c918b12eddedeb97eea0a2d563457e4)) - https://github.com/mojaloop/central-ledger/releases/tag/v17.0.4

Its possible that we have missed others. I recommend double checking this list against the issue report for the PI to double double check.

Reply:
@aaronreynoza is looking into this and will update.